streaming/src/main/java/org/apache/carbondata/streaming/CarbonStreamRecordWriter.java

```diff
@@ -186,7 +188,7 @@ private void initializeAtFirstRow() throws IOException {
       compressorName = carbonTable.getTableInfo().getFactTable().getTableProperties().get(
           CarbonCommonConstants.COMPRESSOR);
       if (null == compressorName) {
-        compressorName = CompressorFactory.getInstance().getCompressor().getName();
+        compressorName = CompressorFactory.NativeSupportedCompressor.SNAPPY.getName();
```

Comment:
revert this or use CCC default compressor

Reply:
fixed